.github/workflows/promote_to_production.yaml

```diff
@@ -0,0 +1,84 @@
+name: Promote staging to production
+
+on:
+  workflow_dispatch:
+    inputs:
+      networks:
+        description: 'Which networks will promote?'
```

Comment:
Are we promoting 1 network or several at once?

Reply:
as input you can provide something like this:
- astar
- astar moonbeam moonriver